.npmignore

```diff
@@ -1,6 +1,8 @@
+.*
+appveyor.yml
+spec
 spec-cordova
 spec-plugman
-spec
+integration-tests
 coverage
-src/plugman/defaults.json
```

Comment:
That file is generated during testing IIRC. Can you confirm? Is it excluded anyway due to being ignored by git?

Reply:
It will not be excluded because it is in the `.gitignore` file in this setup. 

From npm docs
```
Use a .npmignore file to keep stuff out of your package. If there’s no .npmignore file, but there is a .gitignore file, then npm will ignore the stuff matched by the .gitignore file. If you want to include something that is excluded by your .gitignore file, you can create an empty .npmignore file to override it. Like git, npm looks for .npmignore and .gitignore files in all subdirectories of your package, not only the root directory.
```

So from how I am reading this, once the `.npmignore` file is created to exclude items that are not excluded by `.gitignore`, npm will use only the `.npmignore` file.

Reply:
Alright, I checked and `defaults.json` is not created anymore since #775. Thus, we should not need it in any of the ignore files anymore.